func/warble.cc

```diff
@@ -1,5 +1,20 @@
 #include "warble.h"
 
+void WarbleImpl::Call(std::string event_function, const google::protobuf::Any* request, google::protobuf::Any* reply) {
```

Comment:
What is supposed to be in here?

Reply:
It's supposed to create the relevant request and reply. just committed the filled in versions